dpgen/generator/lib/gaussian.py

```diff
@@ -4,16 +4,32 @@
 import uuid
 import itertools
 import numpy as np
+import dpdata
 from scipy.sparse import csr_matrix
 from scipy.sparse.csgraph import connected_components
+from scipy.spatial import cKDTree
 try:
     import openbabel
 except ImportError:
     pass
+try:
+    from ase import Atoms
```

Comment:
No idea about how to handle coordinates with periodic boundary conditions... So I use a third party module instead.